src/rttable.c

```diff
@@ -430,7 +442,7 @@ int insertRoute(uint32_t group, int ifx, uint32_t src) {
 *   activated, it's reinstalled in the kernel. If
 *   the route is activated, no originAddr is needed.
 */
-int activateRoute(uint32_t group, uint32_t originAddr, int upstrVif) {
+int activateRoute(uint32_t group, uint32_t originAddr, signed upstrVif) {
```

Comment:
`int` is by default signed and `signed` is just synonym for `int`. It is not common to write `signed` instead of `int`. So what is there reason for this change?